src/lines/Transformer.jl

```diff
@@ -1,15 +1,32 @@
 """
-Transformer(from, to, y, t_ratio)
+```Julia
+    Transformer(from, to, y, t_ratio)
+```
 
-A transformer representation uses the Π model,
 assuming an ideal transformer in series with an admittance.
+The representation uses the Π model.
+
+# Mathematical Representation
+The voltage transforms as:
+```math
+    u_{to} = t_{ratio} u_{from}
+```
+
+# Arguments
+
+- `from` : start node
+- `to` : end node
+- `y`: transformer admittance
+- `t_ratio`: transformation ration
+
+# Assumptions
+
 The admittance is here taken to be on the high-voltage side.
 """
-@Line Transformer(y, t_ratio) begin
+@Line Transformer(from, to, y, t_ratio) begin
```

Comment:
@luap-pik: why did you add the "from" and "to" here when it is unused?

Reply:
Oh it was just annoying in my application that transformers don't follow the way other lines are used. For instance, when I want to do sth like "Give me the voltage  at the `from` node" in a control loop, I need (sketch)

```Julia
...
trafo_idx = findfirst(pg.lines isa Transformer)
from_node = pg.lines[trafo_idx].from
voltage = operationpoint[from_node, :v]
...
```